packages/cli/src/commands/build.ts

```diff
@@ -261,9 +261,15 @@ export async function build({
         )
       );
 
+      console.log(
+        yellow(`Rerunning the same build command will attempt to execute skipped steps. It will not re-run executed steps.`)
```

Comment:
put in a copy-pastable command to rebuild because it actually isn't the same build command

Reply:
Just added this to this PR. Should be good to go.